framework/wcf/Extensibility/Transport/AzureQueueStorageClient/AzureQueueStorageClient/AzureQueueStorageTransportBindingElement.cs

```diff
@@ -0,0 +1,303 @@
+﻿//----------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+//----------------------------------------------------------------
+
+using System;
+using System.ServiceModel.Channels;
+using System.Xml;
+
+namespace Microsoft.Samples.AzureQueueStorage
+{
+    /// <summary>
+    /// AzureQueueStorage Binding Element.  
+    /// Used to configure and construct AzureQueueStorage ChannelFactories and ChannelListeners.
+    /// </summary>
+    public class AzureQueueStorageTransportBindingElement
+        : TransportBindingElement // to signal that we're a transport
+    {
+        bool multicast;
+        static XmlDocument xmlDocument;
+
+        public AzureQueueStorageTransportBindingElement()
+        {
+            this.multicast = UdpDefaults.Multicast;
+        }
+
+        protected AzureQueueStorageTransportBindingElement(AzureQueueStorageTransportBindingElement other)
+            : base(other)
+        {
+            this.multicast = other.multicast;
+        }
+
+        public bool Multicast
+        {
+            get
+            {
+                return this.multicast;
+            }
+
+            set
+            {
+                this.multicast = value;
+            }
+        }
+
+        /*public override IChannelFactory<TChannel> BuildChannelFactory<TChannel>(BindingContext context)
+        {
```

Comment:
You need to implement BuildChannelFactory. You should delete the commented out BuildChannelListener code

Reply:
Got it. Will make the changes.